packages/fx-core/src/plugins/resource/bot/configs/scaffoldConfig.ts

```diff
@@ -113,7 +112,7 @@ export class ScaffoldConfig {
     const rawHostType = context.projectSettings?.pluginSettings?.[PluginBot.PLUGIN_NAME]?.[
       PluginBot.HOST_TYPE
     ] as string;
-    return utils.convertToConstValues(rawHostType, HostTypes);
+    return utils.convertToConstValues(rawHostType, HostType);
```

Comment:
Since we change the type of HostTypes to HostType, do we need to update the method of convertToConstValues() due to the param type is different?